tests/unit/test_debugger.py

```diff
@@ -277,6 +277,61 @@ def test_render_related_event_series(dummy_tables_path):
     ]
 
 
+def test_render_dataset_event_tables_with_population(dummy_tables_path):
```

Comment:
Will it be confusing to users that `show(dataset)` will only show them the patient variables, when `test` is another thing on `dataset`?

Reply:
Yes, I think it will. But we'll need to do some reasonably invasive refactoring of the debug code to get it support returning multiple tables, with some kind of heading for each table to indicate what it is. And that didn't feel like a priority at this stage. I just wanted to make sure it was in principle usable with `show()`.